### PR TITLE
Task 4  components part 2

### PR DESCRIPTION
**Pull request by katefaith**

### CODE REVIEW

File: src/components/Forms/editForm.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './form.scss';
+
+import { filterItems } from '../../mockedData';
+
+export const EditForm = ({ movie }) => (
+  <form className="form">
```
Comment (mmarkelov) [inline src/components/Forms/editForm.jsx]: You just have the same `form` structure in `addForm`. Can you separate it to reuse?

File: src/components/Forms/addForm.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+/* eslint-disable react/button-has-type */
```
Comment (mmarkelov) [inline src/components/Forms/addForm.jsx]: I think it’s better to rename your component from `addForm` to `AddForm`

File: src/components/Header/header.jsx [hunk 3]
```diff
@@ -1,17 +1,35 @@
...
+        </div>
+        <Search />
       </div>
-      <Search />
-    </div>
-  </header>
-);
+
+      {isAddModalOpen && (
+      <Modal setIsModalOpen={setIsAddModalOpen}>
```
Comment (mmarkelov) [inline src/components/Header/header.jsx]: Not an error, but i think that there is more sense to pass function to close modal, like this:
```js
closeModal={() => setIsAddModalOpen(false)}
```

File: src/components/MovieCard/movieCard.jsx [hunk 4]
```diff
@@ -1,27 +1,77 @@
...
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const [isMenuBtnShown, setIsMenuBtnShown] = useState(false);
+  const [isEditModalOpen, setIsEditModalOpen] = useState(false);
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+
+  return (
+    <div
+      className="movie-card"
+      onMouseEnter={() => setIsMenuBtnShown(true)}
+      onMouseLeave={() => { setIsMenuBtnShown(false); setIsMenuOpen(false); }}
```
Comment (mmarkelov) [inline src/components/MovieCard/movieCard.jsx]: also this is not an error, but it's quite difficult to read. It's better to move this kind of callbacks to separate functions:
```js
const onMouseLeave = () => {
  setIsMenuBtnShown(false);
  setIsMenuOpen(false);
}
```

File: src/components/Forms/editForm.jsx (hunk 1 above)
Comment (katefaith) [reply src/components/Forms/editForm.jsx]: Fixed, thanks

File: src/components/MovieCard/movieCard.jsx (hunk 4 above)
Comment (katefaith) [reply src/components/MovieCard/movieCard.jsx]: Fixed, thanks

File: src/components/Header/header.jsx (hunk 3 above)
Comment (katefaith) [reply src/components/Header/header.jsx]: Fixed, thanks

**Pull request merged**